components/wpt-results.html

```diff
@@ -146,40 +153,31 @@
         testFileResults.forEach(result => {
           const testFiles = result.testFiles
           const resultsURL = result.resultsURL
+          const encodedResultsURL = btoa(result.resultsURL)
 
           Object.keys(result.testFiles).forEach(testFileName => {
```

Comment:
Yep that's correct, it's just building the hierarchy of specs. I can add more commenting in a followup.